functions/New-DbaXESmartCsvWriter.ps1

```diff
@@ -0,0 +1,95 @@
+function New-DbaXESmartCsvWriter {
+    <#
+        .SYNOPSIS
+            This Response type is used to write Extended Events to a CSV file.
+
+        .DESCRIPTION
+            This Response type is used to write Extended Events to a CSV file.
+
+        .PARAMETER OutputFile
+            Specifies the path to the output CSV file.
+
+        .PARAMETER OverWrite
+            Specifies whether any existiting file should be overwritten or not.
+
+        .PARAMETER OutputColumn
+            Specifies the list of columns to output from the events. XESmartTarget will capture in memory and write to the target table only the columns (fields or targets) that are present in this list.
+
+            Fields and actions are matched in a case-sensitive manner.
+
+            Expression columns are supported. Specify a column with ColumnName AS Expression to add an expression column (Example: Total AS Reads + Writes)
+
+        .PARAMETER Event
+            Specifies a list of events to be processed (with others being ignored. By default, all events are processed.
+
+        .PARAMETER Filter
+            Specifies a filter expression in the same form as you would use in the WHERE clause of a SQL query.
+
+            Example: duration > 10000 AND cpu_time > 10000
+
+        .PARAMETER EnableException
+            By default, when something goes wrong we try to catch it, interpret it and give you a friendly warning message.
+            This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
+            Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
+
+        .NOTES
+            Website: https://dbatools.io
```

Comment:
Not sure if we did this on the other commands, can add `Tags: XE, XESmartTarget` or something to group all of your commands together. We use the tag with `Find-DbaCommand`.

Reply:
Thanks, I added those tags to all the other commands and will soon commit to this command with the same